### DISCUSSION TITLE
Systemvariable wird in HomeKit nicht aktualisiert

### DISCUSSION THREAD

**Discussion by burningbuell** [thkl/hap-homematic#299] [Q&A] [answered]
Ich habe das Problem, dass eine in Homematic angelegte Systemvariable, welche ich über HAP-Homematic in Apple-Homekit eingebunden habe, nur aktualisiert wird, wenn ich HomeKit öffne.
Kennt jemand das Problem und evtl auch eine Lösung, wie man das im Hintergrund realisiert?

**Comment by thkl** [comment, marked as answer]
https://github.com/thkl/hap-homematic/wiki/variables#trigger

**Reply by burningbuell** [reply]
Prima, Danke....den Teil hatte ich schon gelesen, aber anscheinend doch nicht korrekt umgesetzt.
